fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -636,6 +637,12 @@ class HomeFragment : Fragment() {
 
         homeViewModel.sessionToDelete = null
 
+        requireComponents.appStore.state.wasLastTabClosedPrivate?.also {
```

Comment:
I'm not sure I 100% understood your last comment. Was it not possible to do an observation like this [other example in HomeFragment](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt#719)?

Reply:
ahhhh, right, the rest. Gotcha gotcha. In that case, I'm fine to proceed as-is